### DISCUSSION TITLE
Skip powering on TV on startup/reboot

### DISCUSSION THREAD

**Discussion by djc5166** [JPersson77/LGTVCompanion#180] [Q&A]
Any way to keep the TV from powering at startup/reboot? I only want the TV to turn on when I enable it in my display settings, it is usually disabled unless I am going to play a game, movie, etc. since it is not my main display. Right now it turns on every time I start or reboot my PC.

I've unchecked all the shutdown options:
<img width="163" alt="image" src="https://github.com/JPersson77/LGTVCompanion/assets/19207074/dcb48c5f-1f4d-40e6-aa9a-942c4d841474">

Also disabled setting the HDMI input, since I thought that might also be turning the TV on:
<img width="413" alt="image" src="https://github.com/JPersson77/LGTVCompanion/assets/19207074/99d80567-e4c5-4f20-b659-7536e413e5ca">

"Always Ready" enabled, LG C2 42"

**Comment by JPersson77** [comment]
Hi, the only way currently to achieve that is to disable automatic management of the device, and then enable automatic management of the device when you enable it in the display settings.

Btw, you want to check the "restart" word or the TV will shut down during a reboot.

I can consider adding an option to assume the display topology remains the same on boot as previous session

**Reply by djc5166** [reply]
Sorry not sure what you mean by disabling/enabling 'automatic management' is this a LGTC setting or a windows setting?

Some extra info: I am using monitor profile switcher to switch between monitor settings/layouts (Switching primary/secondary between another 4K monitor and the TV, running only the Monitor, or TV, etc.). The TV turns on/off great with that. I just want to keep it off (if it is already off) during a startup/reboot.

**Comment by JPersson77** [comment]
This is an LGTV Companion setting, which basically enables/disables automatic management of the TV. In the UI it's this checkbox:

![image](https://github.com/JPersson77/LGTVCompanion/assets/77928210/cedb8965-679d-401f-aac0-ea95a67da46b)

Or, alternatively, you can control it via command line;

`"LGTV Companion.exe" -autoenable device1 `

So, what you can do potentially is to disable the checkbox in the UI (click Apply) which will disable automatic management and then either 

a) if your monitor profile switcher program supports it - execute the above command line before switching to a profile which includes the TV, to enable automatic management of the TV.

b) if you monitor profile switcher program has no such support - create a .bat or similar script to execute the above command line and then switch the monitor profile via command line.

This approach will never power on the TV at boot. Not even when the TV is part of the current display topology.

But like I mentioned, I can consider adding an option to assume the display topology remains the same on boot as previous session, which would 
help with your issue

**Reply by djc5166** [reply]
Oh I see what you mean,

https://sourceforge.net/projects/monitorswitcher/

It's pretty simple, I don't think it has any kind of support for running external scripts. This was the really the only thing I found that could easily switch the primary display around or disable/enable displays. Maybe there is a better app out there for this.

I'll play around with what you suggested.

That would be a great feature if it is doable, maybe a bit of a niche use case.

**Comment by JPersson77** [comment]
There are a few who mentioned that they would like the TV to not briefly power on during boot if the TV is not part of the display topology (or more accurately - was not part of the display topology during shutdown). It is designed this way for a reason, but as a user option it might make sense

**Comment by djc5166** [comment]
Seems to be working in v3.2.5, thanks for implementing it.
